### PR TITLE
linting & few fixes

### PR DESCRIPTION
**Pull request by asoorm**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -24,27 +37,23 @@ func init() {
 		Credentials: credentials.NewStaticCredentials("AKID", "SECRET", ""),
 	})
 	if err != nil {
-		fmt.Println("Couldn't get AWS access")
-		panic("Internal Error")
+		log.Fatalf("couldn't get AWS access: %v", err.Error())
```
Comment (sedkis) [inline main.go:42]: Don't we need to return here?

File: main.go [hunk 2]
```diff
@@ -14,7 +15,19 @@ import (
 	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
 )
 
-var svc = &dynamodb.DynamoDB{}
+const (
+	usersTable = "users"
```
Comment (sedkis) [inline main.go:21]: in this specific example, I actually created a table called "basic-auth".  if we change it to "users" the plugin won't work out of the box

File: main.go (hunk 1 above)
Comment (asoorm) [reply main.go:42]: you are causing app to exit with `Fatal`

File: main.go (hunk 2 above)
Comment (asoorm) [reply main.go:21]: this is a PoC - so it doesn't matter what the table is called. either configurable, or hardcode it. but is a draft PR - so was planning on making it configurable - eventually.

File: main.go (hunk 1 above)
Comment (sedkis) [reply main.go:42]: shouldn't it though?

Comment (asoorm) [reply main.go:42]: doesn't need to return.

```
// Fatalf is equivalent to Printf() followed by a call to os.Exit(1).
func Fatalf(format string, v ...interface{}) {
	std.Output(2, fmt.Sprintf(format, v...))
	os.Exit(1)
}
```

Comment (sedkis) [reply main.go:42]: oh that is weird.  Never seen a log function do anything except for print a log.  Is this common in GO?

**Pull request merged**